kombu/transport/SQS.py

```diff
@@ -190,6 +210,14 @@ def _new_queue(self, queue, **kwargs):
 
     def _delete(self, queue, *args, **kwargs):
         """Delete queue by name."""
+        # If we are using predefined queue URLs,
```

Comment:
I don't think that an exception needs to be raised here, because it may be backwards incompatible. An early return would probably suffice.